### PR TITLE
chore: update tabs component copy

### PR DESCRIPTION
**Pull request by rossmoody**
## Description
This PR introduces updates copy from UXCS for the `Tabs` component page. 

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/tabs.mdx [hunk 1]
```diff
@@ -14,13 +14,23 @@ components:
...
     </ul>
   </Use>
   <Misuse>
     <ul>
-      <li>As a secondary navigation bar that spans multiple pages</li>
+      <li>
+        To guide users through a task list, use a{' '}
+        <a href="https://zendeskgarden.github.io/react-components/accordions/#stepper">Stepper</a>{' '}
+        instead
+      </li>{' '}
```
Comment (jzempel) [inline src/pages/components/tabs.mdx:27]: This trailing space shouldn't be needed.

File: src/pages/components/tabs.mdx [hunk 2]
```diff
@@ -14,13 +14,23 @@ components:
...
+      <li>
+        {' '}
+        To help users navigate to markedly different kinds of content, use <a href="/components/anchor">
+          Anchors
+        </a> and link to a separate page{' '}
```
Comment (jzempel) [inline src/pages/components/tabs.mdx:32]: These leading and trailing spaces shouldn't be needed.

Comment (rossmoody) [reply src/pages/components/tabs.mdx:32]: Prettier is drunk

Comment (jzempel) [reply src/pages/components/tabs.mdx:32]: I quite agree

File: src/pages/components/tabs.mdx [hunk 3]
```diff
@@ -14,13 +14,24 @@ components:
...
     <ul>
-      <li>To provide pre-filtered views of a single dataset</li>
-      <li>To organize related content and controls within a page</li>
+      <li>To filter information into easily parsable views</li>
+      <li>To organize related content and controls within a single page</li>
     </ul>
   </Use>
   <Misuse>
     <ul>
-      <li>As a secondary navigation bar that spans multiple pages</li>
```
Comment (allisonacs) [inline src/pages/components/tabs.mdx:23]: This doesn't seem to be encapsulated in any of the new "Not for this" items. Whether or not we're ok with it from a visual design standpoint, the `tab` component has a specific ARIA function, and can't be easily used when the different tabs take you to a different page. Why was this removed? cc @vsri1137

Comment (vsri1137) [reply src/pages/components/tabs.mdx:23]: Hm, I think I didn't understand this properly. That is what I hoped to capture with: 

> To help users navigate to markedly different kinds of content, use [Anchors](#) and link to a separate page.

Maybe we can replace this line with:

As a secondary navigation bar that spans multiple pages. Use Anchors instead.

What do you think?

Comment (allisonacs) [reply src/pages/components/tabs.mdx:23]: That's great @vsri1137 👌
